### ISSUE TITLE
Upgrade axiod to support Deno 1.4.0

### ISSUE THREAD

**Issue by eladhaim22** [roonie007/axiod#4]
**Describe the bug**

Deno 1.4.0 introduced stricter checks on the import and export of types for all users using the --unstable flag. These (presumably) will be integrated with Deno stable in an upcoming release.

See relevant section in release blog post: https://deno.land/posts/v1.4#stricter-type-checks-in-code--unstablecode

**To Reproduce**

Import this module in a file and use deno run --unstable to execute the file.

Note there are several errors:

```
error: TS1371 [ERROR]: This import is never used as a value and must use 'import type' because the 'importsNotUsedAsValues' is set to 'error'.
import {
^
    at https://deno.land/x/axiod@0.20.0-0/mod.ts:3:1
```

The import should work without throwing typescript errors relating to type imports and exports.

**Package & Environment Details**

```
$ deno --version
deno 1.4.6
v8 8.7.220.3
typescript 4.0.3
```

**Comment by eladhaim22** [comment]
I fixed it on local if you want i can open a pr

**Comment by roonie007** [comment]
@eladhaim22 that would be awesome thanks

**Comment by roonie007** [comment]
it's merged #5
